cognee/modules/retrieval/lexical_retriever.py

```
@@ -0,0 +1,121 @@
+from cognee.modules.retrieval.base_retriever import BaseRetriever
+from cognee.shared.logging_utils import get_logger
+from cognee.modules.chunking.models.DocumentChunk import DocumentChunk
+from cognee.modules.storage.utils import get_own_properties
+import asyncio
+import json
+from cognee.modules.retrieval.exceptions.exceptions import NoDataError
+from heapq import nlargest
+logger = get_logger("LexicalRetriever")
+
+class LexicalRetriever(BaseRetriever):
+    _global_chunks: dict[str, Any] = {}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)